jib-cli/src/test/java/com/google/cloud/tools/jib/cli/jar/JarProcessorTest.java

```diff
@@ -18,17 +18,24 @@
 
 import static com.google.common.truth.Truth.assertThat;
 
+import com.google.cloud.tools.jib.api.buildplan.AbsoluteUnixPath;
+import com.google.cloud.tools.jib.api.buildplan.FileEntriesLayer;
+import com.google.cloud.tools.jib.api.buildplan.FileEntry;
 import com.google.cloud.tools.jib.cli.jar.JarProcessor.JarType;
+import com.google.common.collect.ImmutableList;
 import com.google.common.io.Resources;
 import java.io.IOException;
 import java.net.URISyntaxException;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
 import org.junit.Test;
 
 public class JarProcessorTest {
 
   private static final String SPRING_BOOT_RESOURCE_DIR = "jar/springboot/springboot_sample.jar";
+  private static final String STANDARD_RESOURCE_DIR_WITH_CP = "jar/standard/standardJarWithCp.jar";
```

Comment:
```suggestion
  private static final String STANDARD_JAR_WITH_CLASS_PATH_MANIFEST = "jar/standard/standardJarWithCp.jar";
```

Also applies to other constants.